closure/bin/build/jscompiler.py

```diff
@@ -69,8 +69,7 @@ def _JavaSupports32BitMode():
   return supported
 
 
-def _GetJsCompilerArgs(compiler_jar_path, java_version, source_paths,
-                       jvm_flags, compiler_flags):
+def _GetJsCompilerArgs(compiler_jar_path, java_version, jvm_flags):
```

Comment:
You'll need to update the unit test [jscompiler_test.py](https://github.com/google/closure-library/blob/master/closure/bin/build/jscompiler_test.py) to reflect these changes.